.gitignore

```
@@ -0,0 +1,4 @@
+node_modules
+menu
+reservations
+reviews
```

Add new line at end of file
```suggestion
reviews

```